README.md

```diff
@@ -1,6 +1,6 @@
 # <img alt="apalis-board" src="screenshots/logo.svg" width="24px" /> apalis-board
 
-Apalis board contains a nummber of creates useful for building UIs for [apalis](https://github.com/geofmureithi/apalis).
+Apalis board contains a number of creates useful for building UIs for [apalis](https://github.com/geofmureithi/apalis).
```

Comment:
creates -> crates ?

Reply:
you're welcome. 
Also, do you know if this project is still maintained ? I tried to run the frontend project without success.